pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -113,165 +126,181 @@ var _ = Describe("vmexport", func() {
 
 		It("VirtualMachineExport processing fails when using 'download'", func() {
 			const errMsg = "processing failed: Test error"
-
 			vmexport.WaitForVirtualMachineExport = func(_ kubecli.KubevirtClient, _ *vmexport.VMExportInfo, _, _ time.Duration) error {
 				return errors.New(errMsg)
 			}
 
 			err := runDownloadCmd(
-				setFlag(vmexport.PVC_FLAG, "test-pvc"),
+				setFlag(vmexport.PVC_FLAG, pvcName),
 				setFlag(vmexport.OUTPUT_FLAG, filepath.Join(GinkgoT().TempDir(), "disk.img")),
```

Comment:
Maybe also extract `filepath.Join(GinkgoT().TempDir(), "disk.img")` from here?

Reply:
There's already a commit doing that.